src/test/java/nextstep/subway/acceptance/LineSectionAcceptanceTest.java

```diff
@@ -0,0 +1,94 @@
+package nextstep.subway.acceptance;
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static nextstep.subway.acceptance.LineSteps.*;
+import static nextstep.subway.acceptance.StationSteps.지하철역_생성_요청;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DisplayName("지하철 구간 관리 기능")
+class LineSectionAcceptanceTest extends AcceptanceTest {
+    private Long 신분당선;
+
+    private Long 강남역;
+    private Long 양재역;
+
+    /**
+     * Given 지하철역과 노선 생성을 요청 하고
+     */
+    @BeforeEach
+    public void setUp() {
+        super.setUp();
+
+        강남역 = 지하철역_생성_요청("강남역").jsonPath().getLong("id");
+        양재역 = 지하철역_생성_요청("양재역").jsonPath().getLong("id");
+
+        Map<String, String> lineCreateParams = createLineCreateParams(강남역, 양재역);
+        신분당선 = 지하철_노선_생성_요청(lineCreateParams).jsonPath().getLong("id");
+    }
+
+    /**
+     * When 지하철 노선에 새로운 구간 추가를 요청 하면
+     * Then 노선에 새로운 구간이 추가된다
+     */
+    @DisplayName("지하철 노선에 구간을 등록")
+    @Test
+    void addLineSection() {
+        // when
+        Long 정자역 = 지하철역_생성_요청("정자역").jsonPath().getLong("id");
+        지하철_노선에_지하철_구간_생성_요청(신분당선, createSectionCreateParams(양재역, 정자역));
+
+        // then
+        ExtractableResponse<Response> response = 지하철_노선_조회_요청(신분당선);
```

Comment:
```suggestion
        var 정자역 = 지하철역_생성_요청("정자역").jsonPath().getLong("id");
        지하철_노선에_지하철_구간_생성_요청(신분당선, createSectionCreateParams(양재역, 정자역));

        // then
        var response = 지하철_노선_조회_요청(신분당선);
```

가독성과, 더 의미있는 변수 명을 짓기 위해 저는 타입 추론을 자주 사용해요. 요번에 한번 var를 사용해 볼까요?

Reply:
넵~ 타입 추론을 잘 사용하지 않다보니 어색해서 안쓰게되네요 :sweat_smile: 저도 이번부터 조금씩 사용해보도록 하겠습니다 ㅎㅎ